### PR TITLE
doc - bindAndHandleSync doesn't allow parallelism

### PR DESCRIPTION
**Pull request by richardimaoka**
Let me know if I should raise an issue for this. I didn't, since this is just a small correction.

The doc says:

> If you use one of the `Http.bindAndHandleSync` or `Http.bindAndHandleAsync` entry-points, you can specify the `parallelism` argument 

but the `parallelism` argument is only available in the `Http.bindAndHandleAsync` method, not the `Sync` one.

https://github.com/akka/akka-http/blob/3feb8b617bddaae28bf85c08bb2fc1f5c6901c8c/akka-http-core/src/main/scala/akka/http/scaladsl/Http.scala#L292

https://github.com/akka/akka-http/blob/3feb8b617bddaae28bf85c08bb2fc1f5c6901c8c/akka-http-core/src/main/scala/akka/http/javadsl/Http.scala#L342

### CODE REVIEW

**Comment by johanandren** [comment]
OK TO TEST

File: docs/src/main/paradox/server-side/low-level-api.md [hunk 1]
```diff
@@ -158,8 +158,8 @@ HTTP pipelining is generally discouraged (and [disabled by most browsers](https:
 is nevertheless fully supported in Akka HTTP. The limit is applied on two levels. First, there's the
 `akka.http.server.pipelining-limit` config setting which prevents that more than the given number of outstanding requests
 is ever given to the user-supplied handler-flow. On the other hand, the handler flow itself can apply any kind of throttling
-itself. If you use one of the `Http.bindAndHandleSync` or `Http.bindAndHandleAsync`
-entry-points, you can specify the `parallelism` argument (default = 1, i.e. pipelining disabled) to control the
+itself. If you use the `Http.bindAndHandleAsync`
+entry-point, you can specify the `parallelism` argument (default = 1, i.e. pipelining disabled) to control the
```
Comment (ktoso) [inline docs/src/main/paradox/server-side/low-level-api.md]: `bla = 1` syntax reads weird in documentation, this is prose, not code :) Will adjust and merge, thanks!

**Comment by ktoso** [comment]
Thanks :)

**Comment by ktoso** [comment]
Failure was whitesource ignore it.

```
[error] sbtwhitesource.WhiteSourceException: Error communicating with service: Server error: org.springframework.transaction.UnexpectedRollbackException: JTA transaction unexpectedly rolled back (maybe due to a timeout); nested exception is javax.transaction.RollbackException: ARJUNA016053: Could not commit transaction.
[error] 	at sbtwhitesource.CheckPoliciesAction.sendCheckPolicies(WhiteSource.scala:383)
[error] 	at sbtwhitesource.CheckPoliciesAction.doExecute(WhiteSource.scala:365)
[error] 	at sbtwhitesource.BaseAction.execute(WhiteSource.scala:69)
[error] 	at sbtwhitesource.WhiteSourcePlugin$.$anonfun$projectSettings$9(WhiteSourcePlugin.scala:152)
[error] 	at sbtwhitesource.WhiteSourcePlugin$.$anonfun$projectSettings$9$adapted(WhiteSourcePlugin.sca
```

**Pull request merged**